src/components/Icon.vue

```diff
@@ -25,6 +25,9 @@ export default {
       return `vi-icon vi-icon-${this.name}`;
     },
   },
+  mounted() {
+    console.warn('Vi-Icon is deprecated since 0.0.22.');
```

Comment:
Maybe we can display this message only in development mode.

Reply:
The idea is to show this whenever the component is used and is only shown on mout. Putting this in dev mode defeats the purpose of the warning.

Reply:
Take a look: https://vuejs.org/v2/guide/deployment.html

Reply:
![image](https://user-images.githubusercontent.com/713484/47436345-b48f3200-d77c-11e8-9a99-ab69e1b9e4b9.png)